Source/CMake/HelperMethods.cmake

```diff
@@ -192,6 +192,35 @@ MACRO(end_find_package FOLDER_NAME MAIN_LIB_NAME)
 	set(${FOLDER_NAME}_INCLUDE_DIRS ${${FOLDER_NAME}_INCLUDE_DIR})
 ENDMACRO()
 
+MACRO(bs_add_test)
```

Comment:
What's the purpose of this macro vs. using `add_test`? Also, I'm seeing references to `Debug` and `Release` - does it work on other configs?

Reply:
i updated with commit for explanation. Esentially I was getting "found .so library" errors until I updated the tests. plus it's just useful to have add_test wrapper for future in case there's other common properties.

Reply:
I honestly don't know if it works for other configs. But it at least works for my needs at the moment.